src/tasks/approvals.ts

```diff
@@ -6,8 +6,8 @@ import { IAppConfig } from "../interfaces/config/iappconfig";
 
 export default class FourEyePrincipleTask extends BaseTask<any> { 
 
-  constructor(appconfig : IAppConfig, config : any, repo: {repo: string, owner: string}) {
-    super(appconfig, config, repo); 
+  constructor(appconfig : IAppConfig, config : any, repo: {repo: string, owner: string}, organization: string | undefined) {
```

Comment:
are you willing to keep a possible `undefined`state in the code base. What about providing just a default empty string ?

Reply:
So I just checked a bit the TS docs and maybe `organization` should be an optional parameter ? https://www.typescriptlang.org/docs/handbook/functions.html#optional-and-default-parameters

Reply:
hmm yeah, null might be the more correct value here, since all tasks will be loaded in the same way through the task-runner, the optional parameter will always be used.

Reply:
Also, the correct way to go about this would be to have a TaskParams type instead of all these seperate parameters